src/functions/template-tags/general.php

```diff
@@ -1518,48 +1527,60 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null, $skip
 		 */
 		$remove_shortcodes = apply_filters( 'tribe_events_excerpt_shortcode_removal', true );
 
-		// Get the Excerpt or content based on what is available
-		$excerpt = has_excerpt( $post->ID ) ? $post->post_excerpt : $post->post_content;
+		$cache_excerpts_key = implode( ':', [
+			$post->ID,
+			$skip_postdata_manipulation,
+			$allow_shortcodes,
+			$remove_shortcodes,
+			json_encode( $allowed_html )
```

Comment:
* :warning: json_encode() is discouraged. Use wp_json_encode() instead.
* :no_entry_sign: Comma required after last value in array declaration
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 861e1957da7830aba4c528766c1afeb7</sub>